tests/test_check.py

```diff
@@ -107,6 +107,30 @@ def test_check_no_description(monkeypatch, capsys):
     )
 
 
+def test_check_invalid_description(monkeypatch, capsys):
+    package = pretend.stub(metadata_dictionary=lambda: {
+        'description': "Project",
+        'long_description': "My amazing project::",
+        'description_content_type': '"text/rst"',  # close, but not quite
+    })
+
+    monkeypatch.setattr(check, "_find_dists", lambda a: ["dist/dist.tar.gz"])
+    monkeypatch.setattr(
+        check,
+        "PackageFile",
+        pretend.stub(from_filename=lambda *a, **kw: package),
+    )
+
+    output_stream = check.StringIO()
+    check.check("dist/*", output_stream=output_stream)
+    assert output_stream.getvalue() == (
```

Comment:
I mean maybe? I could basically check for the existence of `warning; `long_description_content_type` invalid` to ensure I hit the line. But it would be behaving pretty differently then the other tests of the file

Reply:
I've gotten the impression that there's leeway to deviate from the existing test structures, with this sort of thing being an example. I think your proposed check would be sufficient.